modules/dnn/src/onnx/onnx_importer.cpp

```diff
@@ -2311,10 +2311,10 @@ void ONNXImporter::parseUnsqueeze(LayerParams& layerParams, const opencv_onnx::N
     int axis = axes.getIntValue(0);
     axis = axis < 0 ? axis + (int)inpShape.size() + 1 : axis;
     CV_Assert(0 <= axis && axis <= inpShape.size());
-    std::vector<int> outShape = inpShape;
+    MatShape outShape = inpShape;
```

Comment:
Since we are getting symbolic inference, the shape inference in onnx importer should be dropped, am I right?

Reply:
yes, I removed it. But it means more cases should be handled during the inference itself.